wellregistry/registry/migrations/0004_grant_client_user.py

```diff
@@ -0,0 +1,39 @@
+"""
+After the registry table is created,
+this will grant access to th client login
+"""
+import sys
+
+from django.db import migrations
+from wellregistry.settings import APP_SCHEMA_NAME
+from wellregistry.settings import APP_CLIENT_USERNAME
```

Comment:
Try pulling `from django.conf import settings`.

Reply:
humm, I thought pycharm found all these. I will be doing so. I am just so surprised that I missed so many.

Reply:
Well, it looks like some might be a dirty workspace (need to push) but I still did miss many.